easybuild/framework/easyblock.py

```diff
@@ -1585,7 +1607,7 @@ def sanity_check_step(self, custom_paths=None, custom_commands=None, extension=F
                 found = False
                 for name in xs:
                     path = os.path.join(self.installdir, name)
-                    if os.path.exists(path):
+                    if check_fn(path):
```

Comment:
even worse, this check will now pass on directories that are actually files

this distinction can be important, for example if you want to make sure that a Python package is installed as a zipped egg, i.e., that the `.egg` is a file rather than a directory

so, we really need a separate PR for this